avm/res/compute/virtual-machine/README.md

```diff
@@ -3646,7 +3646,7 @@ extensionDomainJoinConfig: {
     }
 }
 resource kv1 'Microsoft.KeyVault/vaults@2019-09-01' existing = {
-  name: 'adp-[[namePrefix]]-az-kv-x-001'
+  name: 'adp-#_namePrefix_#-az-kv-x-001'
```

Comment:
Please address any change to different modules than azure firewall in a new PR

Reply:
@eriqua new PR is created: https://github.com/Azure/bicep-registry-modules/pull/1096

Reply:
@eriqua Can we resolve the conversation, as the PR #1096 made that changes, so they are already in main and thus not part of this PR anymore